### ISSUE TITLE
Validate on HTMLCollection

### ISSUE THREAD

**Issue by Mrjaco12** [RadLikeWhoa/Countable#40]
The [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementsByClassName) and most browsers document functions such as `getElementsByClassName` and `getElementsByTagName` returning HTMLCollections rather than node lists. Unfortunately this doesn't pass the current validation which only checks for `[object NodeList]`. Something like:

```
  function _validateArguments (elements, callback) {
    var nodes = Object.prototype.toString.call(elements),
        validObject = nodes === '[object NodeList]' || nodes === '[object HTMLCollection]',
        elementsValid = elements && ((validObject && elements.length) || (elements.nodeType === 1)),
        callbackValid = callback && typeof callback === 'function'

    if ('console' in window && 'warn' in console) {
      if (!elementsValid) console.warn('Countable: No valid elements were found')
      if (!callbackValid) console.warn('Countable: "' + callback + '" is not a valid callback function')
    }

    return elementsValid && callbackValid
  }
```

But I'm not sure how the tests would work since they seem to be designed for single elements.

**Comment by RadLikeWhoa** [comment]
Good catch, this is fixed in b73cafb. Thanks.

**Comment by Mrjaco12** [comment]
Awesome thanks!
